src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/HttpVersionUtils.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+
+namespace Microsoft.PowerShell.Commands
+{
+    internal static class HttpVersionUtils
```

Comment:
Given the fact that this is only used on the attribute, the attribute is internal and is only used in a specific context, it feels to me like the functionality in this class just belongs in the attribute class instead